src/Components/TimeSheets/index.js

```diff
@@ -1,123 +1,131 @@
 import { useEffect, useState } from 'react';
-import { Link } from 'react-router-dom';
-import Modal from './Modal/index';
+import Table from '../Shared/Table/Table';
+import { useLocation } from 'react-router-dom';
 import styles from './timeSheets.module.css';
 
-const TimeSheets = (props) => {
+const TimeSheets = () => {
   const [timeSheets, setTimesheet] = useState([]);
+  // eslint-disable-next-line no-unused-vars
   const [showModal, setShowModal] = useState(false);
-  const [timeSheetId, setTimeSheetId] = useState(undefined);
+  // eslint-disable-next-line no-unused-vars
+  const [itemId, setItemId] = useState('');
+  const location = useLocation();
 
-  useEffect(async () => {
+  const getList = async () => {
     try {
-      const response = await fetch(`${process.env.REACT_APP_API_URL}/timesheets/`);
-      const json = await response.json();
-      setTimesheet(json.data);
+      let response = await fetch(`${process.env.REACT_APP_API_URL}/timesheets/`);
+      response = await response.json();
+      setTimesheet(response.data);
     } catch (error) {
       alert('Could not GET TimeSheets.', error);
     }
-  }, []);
-
-  const closeModal = () => {
-    setShowModal(false);
   };
 
-  const fixDate = (date) => {
-    let dateFormated = date.substr(0, 10);
-    return dateFormated;
-  };
+  useEffect(() => {
+    getList();
+  }, []);
 
+  // eslint-disable-next-line no-unused-vars
   const deleteTimeSheet = async (id) => {
     try {
       const response = await fetch(`${process.env.REACT_APP_API_URL}/timesheets/${id}`, {
         method: 'DELETE'
       });
       if (response.status === 204) {
-        alert('TimeSheet removed.');
+        alert('Timesheet removed.');
         setTimesheet([...timeSheets.filter((timeSheet) => timeSheet._id !== id)]);
       } else {
-        alert('TimeSheet could not be removed.');
+        alert('Timesheet could not be removed.');
       }
     } catch (error) {
-      alert('TimeSheet could not be removed.', error);
+      alert('Timesheet could not be removed.', error);
     }
   };
 
+  const timeSheetList = [];
+  for (let i = 0; i < timeSheets.length; i++) {
+    const timeSheet = timeSheets[i];
+    if (timeSheet.task == null && timeSheet.employee == null && timeSheet.projet == null) {
```

Comment:
"project" has a typo (projet), that's why is was not showing right on the table
```suggestion
    if (timeSheet.task == null && timeSheet.employee == null && timeSheet.project== null) {
```

Reply:
fixed